catroweb.yaml

```diff
@@ -658,7 +660,11 @@ components:
       schema:
         $ref: '#/components/schemas/Uuid'
       required: true
-
+    ProgramId:
```

Comment:
maybe rename it to something more meaningful?

maybe something like

similar_to_project(_id)

@HCrane